lib/app/cubit/staking/staking_cubit.dart

```diff
@@ -30,10 +32,23 @@ class StakingCubit extends Cubit<StakingState> {
     final List<StakingItemViewModel> stakingItemViewModels = [];
     final List<DelegateValidatorItemViewmodel> validatorItemViewModels = [];
 
-    final result =
-        await _getValidatorUseCase.call(const GetValidatorUseCaseParam());
+    final results = await Future.wait([
+      _getValidatorUseCase.call(const GetValidatorUseCaseParam()),
+      _getCoinUseCase.call(GetCoinUseCaseParam(
+          param: const CoinRequest(
+              ids: Chain.digChain, vs_currency: Currency.usd)))
+    ]);
 
-    result.fold((l) {
+    final Either<BaseDigException, ValidatorResponse>
+        getValidatorUseCaseResult =
+        results.first as Either<BaseDigException, ValidatorResponse>;
+    final Either<BaseDigException, List<Market>> _getCoinUseCaseResult =
+        results.last as Either<BaseDigException, List<Market>>;
+
+    num currentPrice = 0;
+    num priceChangePercentage24h = 0;
+
+    getValidatorUseCaseResult.fold((l) {
```

Comment:
you should write it like `getValidatorUseCaseResult.fold((failure) {
      ...
    }, (markets) {
      ...
    });`

Its better reading than `l` `r`